src/main.rs

```diff
@@ -52,11 +78,17 @@ fn main() {
         .values_of("extra")
         .map_or_else(Vec::new, |e| e.collect());
 
-    let result = cargo_clone::clone(method, name, version, &extra);
+    let cloner = cargo_clone::Cloner::default().unwrap();
```

Comment:
It seems like this will panic if it can't determine the current dir.

Reply:
That's true but is this different behavior from what [Command](https://doc.rust-lang.org/std/process/struct.Command.html) will do? I can make `out_dir` (previously `work_dir` `Option<PathBuf>` and if it's not specified it's not set. Is that an improvement?

Reply:
I was just thinking it could print a more friendly error instead of panicking (like the code previously did).

Reply:
Updated